mlflow/store/db_migrations/versions/27a6a02d2cf1_add_model_version_tags_table.py

```diff
@@ -27,8 +27,7 @@ def upgrade():
                     sa.ForeignKeyConstraint(
```

Comment:
@zhidongqu-db I think we're missing the backref on the `SqlModelVersion` table that we're defining in the `models.py` file, right?:

```
    model_version = relationship('SqlModelVersion', foreign_keys=[name, version],
                                 backref=backref('model_version_tags',
                                                 cascade='all'))
```

Reply:
Not an issue (see https://stackoverflow.com/questions/28108500/alembic-create-relationship-in-revision-file)